cli/tokens.go

```diff
@@ -85,12 +86,41 @@ func createToken() *cobra.Command {
 	return cmd
 }
 
+// tokenListRow is the type provided to the OutputFormatter.
+type tokenListRow struct {
+	// For JSON format:
+	codersdk.APIKey `table:"-"`
+
+	// For table format:
+	ID        string    `json:"-" table:"id,default_sort"`
+	LastUsed  time.Time `json:"-" table:"last used"`
+	ExpiresAt time.Time `json:"-" table:"expires at"`
+	CreatedAt time.Time `json:"-" table:"created at"`
+	Owner     string    `json:"-" table:"owner"`
+}
+
+func tokenListRowFromToken(token codersdk.APIKey, usersByID map[uuid.UUID]codersdk.User) tokenListRow {
+	user := usersByID[token.UserID]
+
+	return tokenListRow{
+		APIKey:    token,
+		ID:        token.ID,
+		LastUsed:  token.LastUsed,
+		ExpiresAt: token.ExpiresAt,
+		CreatedAt: token.CreatedAt,
+		Owner:     user.Username,
```

Comment:
I'd actually just like to return `owner` on the response [from `client.Tokens`](https://github.com/coder/coder/pull/6227/files#diff-7635cc0f23df9f9d9df4c4d51662f853bf68efcab977bc53bb03389359fd9000R134), given that we'll also have to use it on the FE as well, but I'd like to gate querying for the user if the requester 1) has requested that all tokens be returned and 2) has permissions to see all tokens).